src/utils/Utils.ts

```diff
@@ -71,4 +71,24 @@ export class Utils {
             })
             .join(' + ');
     }
+    
+    /**
+     * @returns {string}
+     */
+    public static generateIden (): string {
```

Comment:
You don't need this. You can use method `getRandomString` of `RandomGenerator` class.
https://github.com/javascript-obfuscator/javascript-obfuscator/blob/master/src/utils/RandomGenerator.ts#L131